commands/operator-sdk/cmd/test.go

```diff
@@ -49,20 +49,21 @@ func NewTestCmd() *cobra.Command {
 	testCmd.Flags().StringVarP(&crdManifestPath, "crd", "c", "deploy/crd.yaml", "Path to CRD manifest")
 	testCmd.Flags().StringVarP(&opManifestPath, "operator", "o", "deploy/operator.yaml", "Path to operator manifest")
 	testCmd.Flags().StringVarP(&rbacManifestPath, "rbac", "r", "deploy/rbac.yaml", "Path to RBAC manifest")
-	testCmd.Flags().BoolVarP(&verbose, "verbose", "v", false, "Enable verbose go test")
+	testCmd.Flags().SetInterspersed(false)
 
 	return testCmd
 }
 
 func testFunc(cmd *cobra.Command, args []string) {
-	testArgs := []string{"test", testLocation + "/..."}
-	if verbose {
-		testArgs = append(testArgs, "-v")
+	if args[0] != "args" {
+		log.Fatalf("Unrecognized argument `%s`; please use the `args` argument if passing arguments/flags to `go test`", args[0])
```

Comment:
I see that the error message starts with mix case as indicated https://github.com/operator-framework/operator-sdk/issues/380. We should define a convention to follow to make the code style more consistent.

Reply:
Maybe I could make a test for to check for this in travis.